packages/medusa/src/api/routes/store/carts/update-cart.ts

```diff
@@ -89,15 +90,19 @@ export default async (req, res) => {
   const validated = req.validatedBody as StorePostCartsCartReq
 
   const cartService: CartService = req.scope.resolve("cartService")
-  await cartService.update(id, validated)
+  const manager: EntityManager = req.scope.resolve("manager")
 
-  const updated = await cartService.retrieve(id, {
-    relations: ["payment_sessions", "shipping_methods"],
-  })
+  await manager.transaction(async (transactionManager) => {
+    await cartService.withTransaction(transactionManager).update(id, validated)
+
+    const updated = await cartService.retrieve(id, {
```

Comment:
question: Why did you remove the use of `withTransaction` here?

Reply:
Conflict not that well managed 🤘